helpdesk/models.py

```diff
@@ -2128,6 +2122,8 @@ def build_api_field(self):
             attributes["max_digits"] = self.max_length
         elif self.data_type == "list":
             attributes["choices"] = self.get_choices()
+        # elif self.data_type == "datetime":
+        #     attributes["default_timezone"] = getattr(settings, "DEFAULT_TIMEZONE", "UTC")
```

Comment:
Is it useful to keep that commented block?

Reply:
Removed